### DISCUSSION TITLE
Vertical interpolation of UFS netcdf output?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [dtcenter/METplus#1185] [File I/O] [answered]
Is it possible to do vertical interpolation of UFS model output to standard pressure levels via METplus?

**Comment by TaraJensen** [comment, marked as answer]
Ben - the short answer is no.  The longer answer is METplus expects the user to run native grids though a post-processing tool, such as UPP, to interpolate output to standard pressure levels.  Hopefully this helps.

**Reply by benjamin-cash** [reply]
Hi Tara - Alas, I'm attempting to avoid learning UPP in addition to everything else I am currently drowning in. ;) But I think cdo has a very simple option for this so I will give that a shot. Thanks for the quick reply!
